shell/platform/glfw/external_texture_gl.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/glfw/external_texture_gl.h"
+
+#include <glad/gl.h>
+
+// glad.h must be included before glfw3.h.
+#include <GLFW/glfw3.h>
+
+namespace flutter {
+
+// Just to not declare GLFWwindow and GLuint in the header.
```

Comment:
Out of curiosity, why is this an issue? This is an internal header, not part of the API surface. I wouldn't expect GLFW or GL types to be an issue.

Reply:
Because `GLFW/glfw3.h` is imported before `glad/gl.h`, it will cause an error,
please refer to https://www.glfw.org/docs/latest/quick.html

```
If you are using such a library then you should include its header before the 
GLFW header. This lets it replace the OpenGL header included by GLFW 
without conflicts. This example uses glad2, but the same rule applies to 
all such libraries.
```
So I can't include `GLFW/glfw3.h` or other indirect header files in `external_texture_gl.h` and this will cause `glad/gl.h` to throw a compile error in subsequent include.

```
#error OpenGL header already included (API: gl), remove previous include!
```